dev-python/python-xmp-toolkit/python-xmp-toolkit-2.0.1-r2.ebuild

```diff
@@ -29,7 +29,11 @@ RDEPEND="
 	dev-python/pytz[${PYTHON_USEDEP}]
 "
 
-PATCHES=( "${FILESDIR}"/${P}-test.patch )
+PATCHES=(
+	"${FILESDIR}"/${P}-test.patch
+	"${FILESDIR}"/${P}-gif87a-dropped.patch
+	"${FILESDIR}"/${P}-return-type.patch
+)
```

Comment:
Given all patches are test-related, perhaps concatenate them all in `${P}-test.patch`? Using one file saves inodes, and usually space.

Reply:
👍